src/applications/check-in/pre-check-in/tests/e2e/pages/Introduction.js

```diff
@@ -0,0 +1,19 @@
+import Timeouts from 'platform/testing/e2e/timeouts';
+
+class ValidateVeteran {
```

Comment:
should this class be `Introduction` instead of `ValidateVeteran`?

Reply:
Yes, good catch